### PR TITLE
added config_settings to redaction

### PR DESCRIPTION
**Pull request by NikitaPoly**

### CODE REVIEW

**Review by tannermpeterson — CHANGES_REQUESTED** [review]
will make separate PR for this

File: src/mantarray_desktop_app/main_process/process_monitor.py [hunk 1]
```diff
@@ -225,7 +225,10 @@ def _check_and_handle_server_to_main_queue(self) -> None:
             else:
                 raise NotImplementedError(f"Unrecognized shutdown command from Server: {command}")
         elif communication_type == "update_user_settings":
-            new_values = communication["content"]
+            # if content does not exist then check config_settings
+            new_values = (
+                communication["content"] if "content" in communication else communication["config_settings"]
```
Comment (luciipak) [inline src/mantarray_desktop_app/main_process/process_monitor.py:230]: When does this get sent as `communication["config_settings"]`? 
https://github.com/CuriBio/mantarray-desktop-app/blob/def1d998ae422d4f5915e11271132e3d64e8a4bd/src/mantarray_desktop_app/main_process/server.py#L428

File: src/mantarray_desktop_app/main_process/process_monitor.py [hunk 2]
```diff
@@ -225,7 +225,10 @@ def _check_and_handle_server_to_main_queue(self) -> None:
             else:
                 raise NotImplementedError(f"Unrecognized shutdown command from Server: {command}")
         elif communication_type == "update_user_settings":
-            new_values = communication["content"]
+            # if content does not exist then check config_settings
```
Comment (luciipak) [inline src/mantarray_desktop_app/main_process/process_monitor.py:228]: Are you adding this for the test below? if so, why not just keep it content and change the test?

File: tests/server/test_calling_routes.py [hunk 3]
```diff
@@ -1623,7 +1623,7 @@ def test_set_protocols__returns_error_code_with_invalid_well_name(
         assert f"'{well}'" in response.status
 
 
-@pytest.mark.parametrize("test_invalid_ids", [{"B"}, {"B", "C"}])
+@pytest.mark.parametrize("test_invalid_ids", [{"B"}, {"C", "B"}])
```
Comment (luciipak) [inline tests/server/test_calling_routes.py:1626]: Why were these changed?
